t/unlink.t

```diff
@@ -45,9 +45,10 @@ subtest 'unlink on an existing directory' => sub {
 
     $! = 0;
     is( unlink($temp_dir_name), 0, "MOCKED unlink returns 0 files deleted." );
+    my $err_code = $! + 0;
```

Comment:
Are we just capturing this earlier?

Reply:
Yup. :)

Reply:
What happened was that we were accidentally testing the `-f` on the dockerfile.